docs/spelling_wordlist.txt

```diff
@@ -60,9 +62,12 @@ eClock
 endian
 eProsima
 ethernet
+executables
 extensibility
 fastdds
 fastrtpsgen
+Filesfastcdr
+Filesfastrtps
```

Comment:
I don't know where is this coming from

Reply:
It's because of the backslash (\). In rst it's an escape symbol so we have to escape it too.

Reply:
Solved!